extension/lib/settings_wrapper.js

```diff
@@ -164,6 +203,27 @@ class Settings {
 			cb(termDocumentFreq);
 		})
 	}
+
+	/* 
+	Get's list of URLs that have been used for the document freq dictionary. 
+	Stored as dict for O(1) element search & bc can't store as Set()
+	*/
+	getVisitedUrls(cb) {
+		let key = settingKey.VISITED_URLS;
+		chrome.storage.local.get(key, function(settingsDict) {
+			if (chrome.runtime.lastError) {
+				console.log("Failed to retrieve list of visited URLs: " + chrome.runtime.lastError);
+				return;
+			}
+			// Default visitedUrls (dict)
+			let visitedUrls = {}; 
+			if (key in settingsDict) {
+				visitedUrls = settingsDict[key];
+				console.log(visitedUrls);
```

Comment:
Please use debug(), and give an identifying key word(s) so if im a newbie to the code and want to find what created the log statement i can ctrl-f the keyword easily.
E.g. debug("Visited urls = " + ...)

Reply:
(This is actually just missed console.log, will nuke)